src/TodoList.js

```diff
@@ -1,25 +1,11 @@
 import React from "react";
 import TodoListItem from "./todoListItem";
 
-
-const todoList = [{
-    id: 1,
-    title: 'complete assignment'
-},
-{
-    id: 2,
-    title: 'push assignment to Github'
-},
-{
-    id:3,
-    title: 'take a break'
-}]
-
-function TodoList() {
+function TodoList({todoList}) {
     return (
         <ul>
             {todoList.map((item) => {
-                return <TodoListItem key={item.id} todo={item} />
+                return <TodoListItem key={item.id} todo={item.title} />
```

Comment:
Requires changes:
This is a bit confusing to set `todo={item.title}`. I'd recommend assigning the entire item to`todo={item}` since its name refers to the **entire** todo object. Now within  `TodoListItem`, you can still destructure `todo`, but you'll use `todo.title` instead:
```jsx
export default function TodoListItem({ todo }){
    return (
        <li>{todo.title}</li>
    )
}
```
Optionally, you can destructure `title` from `todo`:
```jsx
export default function TodoListItem({ todo: { title } }){
    return (
        <li>{title}</li>
    )
}
```